indico/modules/events/editing/operations.py

```diff
@@ -205,6 +211,13 @@ def create_revision_comment(revision, user, text, internal=False):
     comment = EditingRevisionComment(user=user, text=text, internal=internal)
     revision.comments.append(comment)
     db.session.flush()
+    editable = revision.editable
+    if editable.can_perform_submitter_actions(user):
+        notify_submitter_comment(revision)
+    if editable.can_perform_editor_actions(user):
+        notify_editor_comment(revision)
+    if not editable.can_perform_submitter_actions(user) and not editable.can_perform_editor_actions(user):
```

Comment:
`elif not editable.can_perform_submitter_actions(user):` is enough here, since the other criterion is implied when using elif